reco_utils/common/spark_utils.py

```diff
@@ -1,23 +1,30 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+import os
+
+
 try:
     from pyspark.sql import SparkSession
 except ImportError:
-    pass  # skip this import if we are in pure python environment
+    SparkSession = None  # skip this import if we are in pure python environment
```

Comment:
why are we setting this to None?

Reply:
mainly to avoid IDE warnings about SparkSession not being defined. Either way won't impact the execution side

Reply:
what IDE are you using? I think it this is specific to one IDE we should keep it with pass.

Reply:
leaving this as it was, in case people want to  change all these instances, please go ahead